### PR TITLE
add profile .,#4737

### PR DESCRIPTION
**Pull request by Legnaleugim**
# Week 1:
## Exercise 1:

1. Forked the repository `/semana-1` following Github official documentation [^1]
2. Cloned the forked repository to my local storage.
3. Added a markdown file named as my discord username ( **.,#4737** ) with the content of my github profile **README**.
4. Commited and pushed the changes to the forked repository.
5. Created a pull request from the forked repository to the upstream repository following Github official documentation [^2]. 

## Exercise 2:

1. Made a new folder inside github profiles with my discord user.
2. Moved the markdown file inside the new folder and renamed it to **README.md**.
3. Added an HTML file with the content of my markdown file.
4. Commited and pushed the changes to the forked repository.

## Exercise 3:

1. Added a CSS file and linked it to my html file.
2. Added some styles to make the html page a little more appealing.
3. Commited and pushed the changes to the forked repository.

## Vercel deployment:

https://undefined-shell-semana-1-legnaleugim.vercel.app/

[^1]: https://docs.github.com/en/get-started/quickstart/fork-a-repo
[^2]: https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request-from-a-fork

### CODE REVIEW

File: github-profiles/{.,}-4737/index.html [hunk 1]
```diff
@@ -0,0 +1,128 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Legnaleugim</title>
+    <style>
```
Comment (olezdev) [inline github-profiles/{.,}-4737/index.html:8]: Sería bueno separar el css del html, va a ser mas cómodo para trabajar

**Review by olezdev — COMMENTED** [review]
Muy agradable el fondo del html, revisar comentario para separar el css del html.

**Review by olezdev — APPROVED** [review]
✅ Revisado, esta muy bueno

**Review by nobody-redhawk — APPROVED** [review]
# Recomendaciones
1. Describir paso a paso las actividades requeridas en los ejercicios de las de la semana 1.
1. Usar etiqueta en el `body` de `header` para el encabezado, `main`, `section` y div para etructurar el contenido para tener mayor contro en la estilizacion.
2. Hacer el deploying con vercel y compartir el link en la descripcion para poder ver el resultado mas facilmente.
3. Espero sea de ayuda, éxitos.

**Comment by Jesusqr-11v** [comment]
Quedo bien.

**Review by Jesusqr-11v — APPROVED** [review]
Carpetas en orden y un readme.md bien hecho.

**Pull request merged**